docs/android-startup-scenarios.md

```diff
@@ -0,0 +1,35 @@
+
+# Android Startup Test
+
+## Prereqs
+
+- Ensure `dotnet` is installed and available with the `dotnet` command. [Dotnet Download](https://dotnet.microsoft.com/en-us/download) or [Daily Dotnet Download](https://github.com/dotnet/sdk/blob/main/documentation/package-table.md)
+- Ensure `xharness` is installed and available with the `xharness` command. The current version in use can be found in the `eng/performance/maui_scenarios_android.proj` file at line 7 (under the tag `MicrosoftDotNetXHarnessCLIVersion`), although any recent version should work. [XHarness Install Instructions](https://github.com/dotnet/xharness?tab=readme-ov-file#installation-and-usage).
+- Have an Android app APK available for testing.
+
+## Steps
+
+1. Initialize the environment (note the . for bash):
+
+    ```sh
+    cd src/scripts
```

Comment:
```suggestion
    cd src/scenarios
```

I think this should point to `scr/scenarios` folder instead (https://github.com/dotnet/performance/blob/main/docs/scenarios-workflow.md#linux).